src/odb/src/zutil/util.cpp

```diff
@@ -165,6 +165,16 @@ int makeSiteLoc(int x, double site_width, bool at_left_from_macro, int offset)
   return site_x1 * site_width + offset;
 }
 
+template <typename T>
+bool hasOverflow(T a, T b)
+{
+  if ((b > 0 && a > std::numeric_limits<T>::max() - b)
+      || (b < 0 && a < std::numeric_limits<T>::min() - b)) {

```

Comment:
You want lowest not min (for floating point min is the smallest positive value).